apps/src/applab/setPropertyDropdown.js

```diff
@@ -267,19 +315,32 @@ function stripQuotes(str) {
 
 /**
  * Gets the properties that should be shown in the dropdown list for elements of the given type.
+ * @param {object} block Optional droplet block (will be undefined in text mode)
+ * @param {object} editor Optional droplet editor (will be undefined in text mode)
  * @param {string} elementType
  * @returns {!Array<string>} list of quoted property names
  */
-function getDropdownProperties(elementType) {
-  if (!elementType) {
-    return fullDropdownOptions;
-  }
+function getDropdownProperties(block, editor, elementType) {
```

Comment:
Is there a particular reason you chose to put the optional parameters in front here?

Reply:
No, that should be fixed. Good point.

Reply:
👍